lib/mjml/parser.rb

```diff
@@ -36,10 +36,10 @@ def render
     #
     # @return [String] The result as string
     # rubocop:disable Style/OptionalBooleanParameter: Fixing this offense would imply a change in the public API.
-    def run(in_tmp_file, beautify = true, minify = false, validation_level = 'strict')
+    def run(in_tmp_file, beautify = true, minify = false, validation_level = 'strict', fonts = Mjml.fonts)
       Tempfile.create(['out', '.html']) do |out_tmp_file|
         command = "-r #{in_tmp_file} -o #{out_tmp_file.path} " \
-                  "--config.beautify #{beautify} --config.minify #{minify} --config.validationLevel #{validation_level}"
+                  "--config.beautify #{beautify} --config.minify #{minify} --config.validationLevel #{validation_level} --config.fonts #{fonts.to_json}"
```

Comment:
Can't manage to have test running successfully without an intensive json formatting almost everywhere